app/mailers/veteran_readiness_employment_mailer.rb

```diff
@@ -1,13 +1,12 @@
 # frozen_string_literal: true
 
 class VeteranReadinessEmploymentMailer < ApplicationMailer
-  def build(user, email_addr)
+  def build(user, email_addr, routed_to_cmp)
     email_addr = 'kcrawford@governmentcio.com' if FeatureFlipper.staging_email?
-
     @submission_date = Time.current.in_time_zone('America/New_York').strftime('%m/%d/%Y')
-    @pid = user.participant_id || 'Sent to CM'
-
-    template = File.read('app/mailers/views/veteran_readiness_employment.html.erb')
+    @pid = user.participant_id.to_s
```

Comment:
Is there a reason this needs to be to_s? The mailer is using erb which should cast int to str.

Reply:
Nope. Can't remember exactly at this moment, but I think I was doing some string interpolation in a private method somewhere in a previous revision before I created another template, and rubocop suggested `.to_s` instead. 

Good catch 🎣